sale_payment_mgmt/views/account_payment_view.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record model="ir.actions.act_window" id="action_sale_account_payment">
+        <field name="name">Payments</field>
+        <field name="res_model">account.payment</field>
+        <field
+            name="groups_id"
+            eval="[(4, ref('sales_team.group_sale_salesman')),(4, ref('sales_team.group_sale_salesman_all_leads')),(4, ref('sales_team.group_sale_manager'))]"
```

Comment:
Don't modify existing ones, but create new.

Reply:
Sorry but i don't understand it. Addon create new custom action about payments.

Reply:
Sorry, my fault in this.